support_bundle/get_support_bundle.sh

```diff
@@ -124,10 +124,8 @@ segment_by="${2}"
 }
 
 main() {
-    # Check for supplied namespace, kube context, and flags
```

Comment:
This code block for namespace is not needed as we default the NAMESPACE variable at the top of the script